### PR TITLE
Be more specific for `-l` CLI arguments

### PR DESCRIPTION
**Pull request by Tbaut**

### CODE REVIEW

File: parity/cli/mod.rs [hunk 1]
```diff
@@ -800,7 +800,7 @@ usage! {
 
 			ARG arg_logging: (Option<String>) = None, or |c: &Config| c.misc.as_ref()?.logging.clone(),
 			"-l, --logging=[LOGGING]",
-			"Specify the logging level. Must conform to the same format as RUST_LOG.",
+			"Specify the general  logging level (error, warn, info, debug or trace). It can also be set for a specific module. Example: '-l sync=debug, rpc=trace',
```
Comment (sorpaas) [inline parity/cli/mod.rs]: Missing `"` in the end, and extra space after `general`. ;)

Comment (Tbaut) [reply parity/cli/mod.rs]: fixed, that was obviously written too quickly :)

File: parity/cli/mod.rs [hunk 2]
```diff
@@ -800,7 +800,7 @@ usage! {
 
 			ARG arg_logging: (Option<String>) = None, or |c: &Config| c.misc.as_ref()?.logging.clone(),
 			"-l, --logging=[LOGGING]",
-			"Specify the logging level. Must conform to the same format as RUST_LOG.",
+			"Specify the general logging level (error, warn, info, debug or trace). It can also be set for a specific module. Example: '-l sync=debug, rpc=trace'",
```
Comment (andresilva) [inline parity/cli/mod.rs]: `, example: ` ?

File: parity/cli/mod.rs [hunk 3]
```diff
@@ -800,7 +800,7 @@ usage! {
 
 			ARG arg_logging: (Option<String>) = None, or |c: &Config| c.misc.as_ref()?.logging.clone(),
 			"-l, --logging=[LOGGING]",
-			"Specify the logging level. Must conform to the same format as RUST_LOG.",
+			"Specify the general logging level (error, warn, info, debug or trace). It can also be set for a specific module, example: '-l sync=debug, rpc=trace'",
```
Comment (5chdn) [inline parity/cli/mod.rs]: Does it really work with space between the options?

Comment (5chdn) [reply parity/cli/mod.rs]:
```
parity -l sync=debug, rpc=trace
error: Found argument 'rpc=trace' which wasn't expected, or isn't valid in this context

USAGE:
    parity [SUBCOMMAND]

For more information try --help

```

**Comment by 5chdn** [comment]
Stop adding F-Labels :rofl:

File: parity/cli/mod.rs (hunk 3 above)
Comment (andresilva) [reply parity/cli/mod.rs]: Damn I tested the example but didn't copy paste it so didn't catch the space.

**Pull request merged**